HomeWork/src/main/java/animals/Cat.java

```diff
@@ -0,0 +1,28 @@
+package animals;
+
+public class Cat extends Animal {
+
+    private static int catsCounter;
+
+    public Cat(String name, int runRestriction, int swimRestriction) {
+        super(name, runRestriction, swimRestriction);
+        catsCounter++;
+    }
+
+    public static int getCatsCounter() {
+        return catsCounter;
+    }
+
```

Comment:
Method `swim()` should be overridden for a cat and print a message `"Cats cannot swim!"`